### PR TITLE
Drag from one source, to anywhere in node. [#93635084]

### PR DESCRIPTION
**Pull request by knowuh**
https://www.pivotaltracker.com/story/show/93635084

Improve link style, make nodes look like PSDs.
Clean up the default drawing a bit
Quote URL in background-image
Small stylus tweaks
Get closer to PSD's for nodes.

### CODE REVIEW

**Comment by knowuh** [comment]
@dougmartin Please have a look.  

The bulf of this is CSS changes and SVG files.

File: src/code/utils/js-plumb-diagram-toolkit.coffee [hunk 1]
```diff
@@ -54,7 +79,7 @@ module.exports = class DiagramToolkit
       @kit.reset()
       @registerListeners()
     else
-      console.log 'No kit defined'
+      console.log "No kit defined"
```
Comment (dougmartin) [inline src/code/utils/js-plumb-diagram-toolkit.coffee]: Should this be converted to use log()?

**Comment by dougmartin** [comment]
Looks good other than converting console.log() to log().

**Comment by knowuh** [comment]
Nice catch on the log().

**Pull request merged**